Makefile

```diff
@@ -66,11 +66,15 @@ integration: deps-cgo ## Run integration test
 	$(DOCKER_CMD) go test -v sigs.k8s.io/cluster-api-provider-libvirt/test/integration
 
 .PHONY: e2e
-e2e: ## Run end-to-end test
-	hack/packet-provision.sh install
-	#TODO run tests
+e2e: e2e-provision ## Run end-to-end test
+	# TODO @ingvagabund @spangenberg add e2e test command here
 	hack/packet-provision.sh destroy
 
+.PHONY: e2e-provision
+e2e-provision: /tmp/packet_ip
```

Comment:
Why can't this be just: 

    e2e-provision:
            hack/packet-provision.sh install

It's not clear what the intermediate target `/tmp/packet_ip` is for.